docs/settings.rst

```diff
@@ -319,5 +319,12 @@ not to do so.
    being built simultaneously. It makes little sense to enable it on
    a workstation.
 
+.. opt:: docker-insecure-registries
 
+   (default ``[localhost]``) A list of docker insecure registries.
```

Comment:
What is insecure registry? Does it allow no-TLS access?

Reply:
Yes, insecure registries don't support `https`.

Reply:
Yeah, I mean can you add a tiny clarification to the docs?